extmod/moducryptolib.c

```diff
@@ -44,10 +44,21 @@ enum {
     UCRYPTOLIB_MODE_MIN = 0,
     UCRYPTOLIB_MODE_ECB,
     UCRYPTOLIB_MODE_CBC,
+#if MICROPY_PY_UCRYPTOLIB_CTR
+    UCRYPTOLIB_MODE_CTR = 6,
+#endif
     UCRYPTOLIB_MODE_MAX,
 };
 
+struct ctr_params {
+    // nonce is the IV of the AES context.
+
+    size_t nc_off;
```

Comment:
What is nc_off?

Reply:
Hmm the names just match the `mbedtls_aes_crypt_ctr` function parameters. I'll use a better name.

Reply:
I rather keep it should for simplicity. Added a comment.

Reply:
Renamed to `offset`.